packages/@uppy/tus/src/index.test.js

```diff
@@ -0,0 +1,28 @@
+const Core = require('@uppy/core')
+const Tus = require('.')
+
+describe('Tus', () => {
+  it('Throws errors if autoRetry option is true', () => {
+    const uppy = new Core()
+
+    expect(() => {
+      uppy.use(Tus, { autoRetry: true })
+    }).toThrowError(/The `autoRetry` option was deprecated and has been removed/)
+  })
+
+  it('Throws errors if autoRetry option is false', () => {
+    const uppy = new Core()
+
+    expect(() => {
+      uppy.use(Tus, { autoRetry: false })
+    }).toThrowError(/The `autoRetry` option was deprecated and has been removed/)
+  })
+
+  it('Throws errors if autoRetry option is `undefined`', () => {
```

Comment:
It’s probably not hurting, but not sure we need tests for a deprecation warning, since we’ll remove it in the future :) Not a problem, just thinking out loud about how we should approach this in the future.

Reply:
My thinking was having tests would help with the code review, I agree such tests doesn't bring a lot of value. Do you want me to remove them?

Reply:
We can keep them since they are already made but perhaps choose not to make tests like this again for deprecation warnings in the future

Reply:
Yep, agreed!